src/test/groovy/graphql/validation/rules/SubscriptionRootFieldNoSkipNoIncludeTest.groovy

```diff
@@ -0,0 +1,151 @@
+package graphql.validation.rules
+
+import graphql.parser.Parser
+import graphql.validation.SpecValidationSchema
+import graphql.validation.ValidationError
+import graphql.validation.Validator
+import spock.lang.Specification
+
+class SubscriptionRootFieldNoSkipNoIncludeTest extends Specification {
+
+    def "valid subscription with @skip and @include directives on subfields"() {
+        given:
+        def query = """
+        subscription MySubscription(\$bool: Boolean = true) {
```

Comment:
This is what I mean by default values. When an empty coerced variables map is passed, this test fails